samples/subsys/display/lvgl/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2023 Benjamin Cabé <benjamin@zephyrproject.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
@kartben This PR has been sitting around for a bit now  - sorry for that. Can you maybe add the conditional select for the `button_matrix` based on `LV_Z_KEYPAD_INPUT`?

Reply:
No worries at all! I will try to update but TBH it also feels like reinstating a "minimal" hello world would also be beneficial now that the original sample has grown into a somewhat rich input devices showcase